### PR TITLE
Front end cleanup

### PR DESCRIPTION
**Pull request by timabell**
This isn't perfect but it's something we can build further on.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ It can also worked on Visual Studio Code but a reverse proxy from https back to
 
 It should be [12 factor](https://12factor.net/), notably config from environment variables.
 
+### Build dependencies
+
+* Nodejs https://nodejs.org/ for the asset pipeline - worked with version 8.11.2 LTS
```
Comment (defong) [inline Readme.md]: Can you add engine for nodejs

https://docs.npmjs.com/files/package.json

File: Readme.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ It should be [12 factor](https://12factor.net/), notably config from environment
 
 * Nodejs https://nodejs.org/ for the asset pipeline - worked with version 8.11.2 LTS
 
+#### Asset pipeline
+
+* See https://docs.microsoft.com/en-us/aspnet/core/client-side/using-grunt?view=aspnetcore-2.1 for how grunt is used.
+* To get visual studio to download the node dependencies open the package.json file and save it. Take a look at the output window (select "Bower/npm" in the output dropdown).
+* Once you've done that you can right-click gruntfile.js in solution explorer and open "Task Runner Explorer" to see that it's been understood.
+* Run a build and it should generate all the files needed under folder wwwroot.
```
Comment (defong) [inline Readme.md]: Can you add additional pipeline to clean/del existing assets before regenerating it altogether

File: Readme.md (hunk 1 above)
Comment (timabell) [reply Readme.md]: Done

File: Readme.md (hunk 2 above)
Comment (timabell) [reply Readme.md]: New PR? This one is already pretty large given the deletions

File: Readme.md (hunk 1 above)
Comment (defong) [reply Readme.md]: Nice one

File: Readme.md (hunk 2 above)
Comment (defong) [reply Readme.md]: OK then.

**Pull request merged**